### ISSUE TITLE
Why is my limit result wrong?

### ISSUE THREAD

**Issue by cyt-codemaker** [sympy/sympy#21310]
```python
from sympy import *
x = Symbol('x')
y = x*(1//x)
print(limit(y,x,10**(-100)))
print(limit(y,x,-10**(-100)))
print(limit(y,x,0))
print(limit(y,x,0,'+'))
print(limit(y,x,0,'-'))
```

`x->0`, the result of `limit(y,x,0) = zoo` while` limit(y,x,10**(-100)) = limit(y,x,-10**(-100)) = 1`. But the limit of this function should be 1. What's wrong with it?

**Comment by oscarbenjamin** [comment]
What version of sympy are you using?

I get:
```
In [3]: from sympy import *
   ...: x = Symbol('x')
   ...: y = x*(1//x)
   ...: print(limit(y,x,10**(-100)))
   ...: print(limit(y,x,-10**(-100)))
   ...: print(limit(y,x,0))
   ...: print(limit(y,x,0,'+'))
   ...: print(limit(y,x,0,'-'))
1.00000000000000
1.00000000000000
1
1
1
```

**Comment by cyt-codemaker** [comment]
> What version of sympy are you using?
> 
> I get:
> 
> ```
> In [3]: from sympy import *
>    ...: x = Symbol('x')
>    ...: y = x*(1//x)
>    ...: print(limit(y,x,10**(-100)))
>    ...: print(limit(y,x,-10**(-100)))
>    ...: print(limit(y,x,0))
>    ...: print(limit(y,x,0,'+'))
>    ...: print(limit(y,x,0,'-'))
> 1.00000000000000
> 1.00000000000000
> 1
> 1
> 1
> ```

Win10, anaconda python 3.8.3, sympy version 1.6.1

**Comment by oscarbenjamin** [comment]
This is fixed in the latest release (1.8) so update your sympy version.

**Comment by cyt-codemaker** [comment]
> This is fixed in the latest release (1.8) so update your sympy version.

OK, thank you for your help!!! (^_^)
